### DISCUSSION TITLE
Failed URI

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#298] [General]
[https://www.biostarhandbook.com/bio-statistics.html - **The four aspects of statistics**](https://www.biostarhandbook.com/bio-statistics.html#:~:text=Our%20favorite%20summary%20for%20statistics%20comes%20from%20the%20blog%20post%20The%20four%20aspects%20of%20statistics%20where%20Frederick%20J.%20Ross%20writes%3A)

Please check status: [https://madhadron.com/posts/2016-01-15-aspects_of_statistics.html](https://madhadron.com/posts/2016-01-15-aspects_of_statistics.html)

It seems necessary to update the link to `https://www.madhadron.com/science/four_aspects_of_statistics.html`

**Comment by ialbert** [comment]
Looks like the link was moved to

https://www.madhadron.com/science/four_aspects_of_statistics.html

the change was made in the source and will be published in the next refresh

**Comment by TullyMonster** [comment]
[https://www.biostarhandbook.com/biological-data-sources.html - **Project systematic names**](https://www.biostarhandbook.com/biological-data-sources.html#project-systematic-names:~:text=detailed%20documentation%20on-,naming%20conventions,-for%20every%20element)

Please check status: http://www.yeastgenome.org/help/community/nomenclature-conventions

It seems necessary to update the link to `https://sites.google.com/view/yeastgenome-help/community-help/nomenclature-conventions`.

**Comment by ialbert** [comment]
Alas it shows how brittle web based information is, they can just do away with entire helpful pages for seemingly little reason.

The fix has been made to link to the WIki

**Comment by TullyMonster** [comment]
[https://www.biostarhandbook.com/human-and-mouse-genomes.html - **How do we transfer genomic coordinates between builds?**](https://www.biostarhandbook.com/human-and-mouse-genomes.html#:~:text=liftOver%20from%20UCSC%20(web%20and%20command%20line))

Please check correspondence: [https://www.ncbi.nlm.nih.gov/genome/tools/remap](https://www.ncbi.nlm.nih.gov/genome/tools/remap)

> It seems necessary to update the link to `https://genome.ucsc.edu/cgi-bin/hgLiftOver`.
> And [remap](https://www.ncbi.nlm.nih.gov/genome/tools/remap) from NCBI (web) will be retired.
